tests/datasets/test_streaming_remote.py

```diff
@@ -0,0 +1,107 @@
+import pathlib
+import time
+
+import pytest
+from PIL.Image import Image
+from torch.utils.data import DataLoader
+
+from composer.datasets.ade20k import StreamingADE20k
+from composer.datasets.utils import pil_image_collate
+
+
+# @pytest.mark.skip()
```

Comment:
@ravi-mosaicml could you provide me a quick example of how to add that @pytest.mark.integration thing? From what I understood I should add **both** the integration flag and the skip(), with the latter being removed once we have later updated our Jenkins workflow to recognize the integration.

Reply:
Can you add in the integration marker in this PR? Here's how:

1. Modify `pyproject.toml` -- it'll need to be specified on line 58 (e.g. `and not integration`) and in the list on line 63

  https://github.com/mosaicml/composer/blob/23f888e07456d31a2950db9f962f4dbdf41938fb/pyproject.toml#L58-L64

2. In `meta.yaml`, modify the existing markers there to include `and not integration`

  https://github.com/mosaicml/composer/blob/97fbab157ebd8c2488f5a00a806e64639df85d19/meta.yaml#L82-L83

3. In `.ci/Jenkinsfile`, modify the markers on line 194 to also include `and not integration`

https://github.com/mosaicml/composer/blob/97fbab157ebd8c2488f5a00a806e64639df85d19/.ci/Jenkinsfile#L194

Then, the `@pytest.mark.skip` will not be needed.

Reply:
Done!